### DISCUSSION TITLE
登陆的时候 显示这个请问什么问题

### DISCUSSION THREAD

**Discussion by jiujiu626** [ZuodaoTech/everyone-can-use-english#1246] [General]
<img width="439" alt="截屏2024-12-12 17 01 52" src="https://github.com/user-attachments/assets/0e4c153a-9a42-4538-b939-31dfff026349" />

**Comment by jiujiu626** [comment]
<img width="433" alt="截屏2024-12-12 17 38 20" src="https://github.com/user-attachments/assets/f34c21e6-5d15-4ad4-9c27-cfb32d9f8731" />
请问我设置了openid密钥，但是想新建智能对话的时候，显示这个报错是什么原因，有没有解决办法

**Reply by an-lee** [reply]
就是字面意思，你的 密钥 用不了 `gpt-4o` 模型
